docs/tutorials/autox/autox.md

```diff
@@ -0,0 +1,198 @@
+    ```
+
+    We can see in the sample output that the experiment has completed and all SLOs and conditions were satisfied.
+
+And the following command allows you to check the results of the experiment.
+
+```bash
+iter8 k report -g autox-httpbin-iter8
+```
+
+??? note "Sample output from report"
+    ```
+    Experiment summary:
+    *******************
+
+    Experiment completed: true
+    No task failures: true
+    Total number of tasks: 4
+    Number of completed tasks: 4
+
+    Whether or not service level objectives (SLOs) are satisfied:
+    *************************************************************
+
+    SLO Conditions                 | Satisfied
+    --------------                 | ---------
+    http/error-count <= 0          | true
+    http/latency-mean (msec) <= 50 | true
+    
+
+    Latest observed values for metrics:
+    ***********************************
+
+    Metric                     | value
+    -------                    | -----
+    http/error-count           | 0.00
+    http/error-rate            | 0.00
+    http/latency-max (msec)    | 25.11
+    http/latency-mean (msec)   | 5.59
+    http/latency-min (msec)    | 1.29
+    http/latency-p50 (msec)    | 4.39
+    http/latency-p75 (msec)    | 6.71
+    http/latency-p90 (msec)    | 10.40
+    http/latency-p95 (msec)    | 13.00
+    http/latency-p99 (msec)    | 25.00
+    http/latency-p99.9 (msec)  | 25.10
+    http/latency-stddev (msec) | 4.37
+    http/request-count         | 100.00
+    ```
+
+    In the sample output, we can see an experiment summary, a list of SLOs and whether they were satisfied or not, as well as any additional metrics that were collected as part of the experiment.
+
+You can also produce an HTML report that you can view in the browser.
+
+```bash
+iter8 k report -g autox-httpbin-iter8 -o html > report.html
+```
+
+??? note "Sample output from HTTP report"
+    ![HTML report](images/htmlreport.png)
+
+##### Continuous and automated experimentation
+
+Now that AutoX is watching the `httpbin` deployment, a new version of this deployment will relaunch the HTTP SLO validation test. However, the version update must be accompanied by a change to its *version label* (specifically, the `app.kubernetes.io/version` label).
+
+For simplicity, we will simply change the version label to the deployment in order to relaunch the HTTP SLO validation test. In the real world, a new version would typically involve a change to the deployment spec (e.g., the container image) and this change should be accompanied by a change to the version label.
+
+```bash
+kubectl label deployment httpbin app.kubernetes.io/version=2.0.0 --overwrite
+```
+
+## Observe new experiment
+
+Check if a new experiment has been launched. Refer to [Observe experiment](#observe-experiment) for the necessary commands.
+
+If we were to continue to update the deployment (and change its version label), then AutoX would relaunch the experiment for each such change.
+
+## Next steps
```

Comment:
We use a note "Some variations and extensions of this experiment" in other tutorials. Perhaps the same here?  See, for example, http://localhost:8000/latest/tutorials/load-test-grpc/